src/components/ErrorBoundary/__tests__/prepareErrorStack.test.ts

```diff
@@ -26,19 +26,8 @@ describe('prepareErrorStack', () => {
     at recoverFromConcurrentError (/static/js/bundle.js:107887:24)
     at performConcurrentWorkOnRoot (/static/js/bundle.js:107800:26)`;
 
-        // Mock window.location.origin that is used inside prepareErrorStack
-        const windowSpy = jest.spyOn(window, 'window', 'get');
-        windowSpy.mockImplementation(() => {
```

Comment:
Newer version of `jest-environment-jsdom` does not allow to manipulate `window` manually. So I had to replace all `window` mocks with different approach.

This test now expects `localhost:3000` base url from config